Donation-platform/src/main/java/com/google/Charity.java

```diff
@@ -29,17 +29,21 @@
     private double userRating;
     private double charityNavRating;
     private double trendingScore;
+    private String link;
+    private String imgSrc;
 
     /** 
      * Constructor takes in details from the database about the charity
      * including name, description, tags, ratings
      */
-    public Charity(String name, String description, Collection<String> tags, double userRating, double charityNavRating) {
+    public Charity(String name, String description, Collection<String> tags, double userRating, double charityNavRating, String link, String imgSrc) {
         this.name = name;
         this.description = description;
         this.tags = tags;
         this.userRating = userRating;
         this.charityNavRating = charityNavRating;
+        this.link = link;
+        this.imgSrc = "images/" + imgSrc;
```

Comment:
Alternatively, we could look at using a link to display the images so we don't have to store that in our app.

Reply:
I actually am not able to get the images to work, because I have a problem with downloading images. I tried to store a direct link to the web where a picture is located and that does not seem to be working yet, either. Will update if I get it fixed, otherwise we can leave that out for the demo, since it's not essential.

Reply:
I think its good to store it in our app. Storing the link might result error tomorrow if someone changes the path of image or remove that image.